package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "acousticbrainz-server",
+  "version": "0.0.0",
+  "description": "package.json for keeping track of nodejs dependencies for acousticbrainz-server",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/metabrainz/acousticbrainz-server.git"
+  },
+  "dependencies": {
+    "babel-core": "6.2.1",
+    "babel-plugin-transform-react-constant-elements": "6.1.20",
+    "babel-plugin-transform-react-inline-elements": "6.1.20",
+    "babel-plugin-transform-react-jsx": "6.2.0",
+    "babel-preset-es2015": "6.1.18",
+    "babel-preset-stage-2": "6.1.18",
+    "babelify": "7.2.0",
+    "bootstrap": "3.3.6",
+    "gulp": "3.9.0",
+    "gulp-less": "3.0.5",
+    "gulp-rev": "6.0.1",
+    "gulp-streamify": "1.0.2",
+    "highcharts": "4.2.0",
+    "jquery": "2.1.4",
+    "less-plugin-clean-css": "1.5.1",
+    "leven": "2.0.0",
```

Comment:
You should be able to remove leven, po2json, and gulp-watch from here (as well as the corresponding imports in the gulpfile), since they aren't used anywhere.